packages/server-wallet/src/protocols/process-ledger-queue.ts

```diff
@@ -315,19 +311,25 @@ export const getProcessLedgerQueueProtocolState = async (
   const ledgerRequests = await store.getPendingLedgerRequests(ledgerChannelId, tx);
   return {
     fundingChannel: await store.getChannel(ledgerChannelId, tx),
-    channelsRequestingFunds: await Promise.all(
-      compose(
-        map(({channelToBeFunded}: LedgerRequestType) => store.getChannel(channelToBeFunded, tx)),
-        filter(['status', 'pending']),
-        filter(['type', 'fund'])
-      )(ledgerRequests)
+    channelsRequestingFunds: _.sortBy(
```

Comment:
Suggestion: Define a helper function `sortByNonce = channelStates => _.sortBy(channelStates, a => a.latest.channelNonce)`, and make this
`await Promise.all(compose(//snip)).then(sortByNonce)`

Reply:
11508ec0f